etc/config/c++.amazon.properties

```diff
@@ -7,6 +7,9 @@ needsMulti=false
 buildenvsetup=ceconan
 buildenvsetup.host=https://conan.compiler-explorer.com
 
+externalparser=CEAsmParser
+externalparser.exe=/usr/local/bin/asm-parser
```

Comment:
It might be nice to install to `/opt/...` and squash like the compilers etc -- if only as then I can use the same `amazon` configuration locally (which is how I test locally). Right now I can use all the same `ce_install` scripts as the main site to install compilers etc. I realise this is quite an ask (and would mean support in ce_install to install things and do stuff like symlink "current" etC).

Reply:
It has an install, `ce_install install asm-parser` (https://github.com/compiler-explorer/infra/blob/main/bin/yaml/tools.yaml#L36) which will place it `/opt/compiler-explorer/asm-parser/asm-parser`

But the init script is currently responsible for making a local copy https://github.com/compiler-explorer/infra/blob/main/init/start.sh#L86

I wanted to make sure it's not "up-to-date always", but a conscious choice to upgrade. I don't think the current test-suite for asm-parser is good enough to make it easier at the moment.

Plus I didn't want to have to risk it being launched from EFS while its only x kb and lose time on latency and undo the extra performance it was supposed to give.

It might require some more thinking

Reply:
Understood! Thanks for the info. I can symlink it locally when testing.

We can also use a "latest" style symlink to make it still an explicit update operation to a named version (and/or put the version here in the c++.amazon.properties, like `/opt/compiler-explorer/asm-parser/v1.2.3/asm-parser` or whatever too